### ISSUE TITLE
Crash on world load NullPointerException RenderLastWorldEvent

### ISSUE THREAD

**Issue by Lehren** [TeamPneumatic/pnc-repressurized#416]
Hello I'm getting this error just when I created a new world and trying to load in.

```
java.lang.NullPointerException
    at me.desht.pneumaticcraft.client.render.pneumatic_armor.HUDHandler.renderWorldLastEvent(HUDHandler.java:75)

```
https://paste.dimdev.org/xomepidada.mccrash

**Comment by desht** [comment]
Can you try build 384 from https://jenkins.k-4u.nl/job/PneumaticCraft-Repressurized/ and let me know if it fixes your problem?

**Comment by Lehren** [comment]
That fixed it thanks

**Comment by desht** [comment]
Fixed in 0.11.10 release
